### ISSUE TITLE
Spyder - debug does not work 

### ISSUE THREAD

**Issue by stefaniecg** [spyder-ide/spyder#19585]
### What steps will reproduce the problem?

i cannot debug, it just does not work, in Spyder version 
* Spyder version: 5.1.5 None
* Python version: 3.9.12 64-bit
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Darwin 21.5.0

**Comment by ccordoba12** [comment]
Hey @stefaniecg, thanks for reporting. This problem is should be fixed in our latest version. Please update by closing Spyder, opening the Anaconda Prompt and running there

    conda update anaconda
    conda install spyder=5.3.3

Let us know if it works for you.

**Comment by stefaniecg** [comment]
Hola @ccordoba12 Carlos, buenas !

I have been able to execute the first command
`conda update anaconda`

but the second: `conda install spyder=5.3.3` just stays hanging in 
`Examining conflict for babel yapf three-merge cryptography sortedcollections`
i've been waiting for an hour and it does not move forward. 

Is there another bug in there that prevents this from finishing?

**Comment by ccordoba12** [comment]
That's a problem with Anaconda, not with Spyder. You have two options to try to solve it:

1. Remove Spyder and reinstall it. You can do that by closing Spyder and running the following commands in the Anaconda Prompt:

        conda remove spyder
        conda install spyder

    You need to be aware that this command can remove more packages than just Spyder (e.g. Matplotlib) so you'll need to reinstall them too.

2. Remove and reinstall Anaconda again. That's more radical but it should be easier to do. After doing that, you need to run

    conda update anaconda
    conda install spyder=5.3.3

**Comment by stevetracvc** [comment]
I prefer to have a conda environment just for spyder, since there can be a bunch of dependencies that don't play well with some of my other environments. Then, any conda environment that you want to use in spyder will need the spyder-kernels package installed too.

https://docs.spyder-ide.org/current/installation.html

**Comment by ccordoba12** [comment]
Right, that's a good idea too. Furthermore, we're working right now to create conda-based apps that work precisely like this (see PR #19461).

**Comment by stefaniecg** [comment]
Thank you guys for all your comments and ideas.
Based on the post above by @ccordoba12:

**Method 2**:
Reinstalling Anaconda did not work.
I had the same problem as before.

**Method 1**
So I tried the first method, which is uninstalling spyder and intalling it again.
That worked, i have the last version _spyder=5.3.3_, but many other packages were uninstalled as mentioned above.
This is annoying, as i have to install basic packages now one by one. 

So, my next question is: _is there a command to install all basic python packages_?

**Comment by ccordoba12** [comment]
Unfortunately there isn't, sorry.

**Comment by ccordoba12** [comment]
But is the debugger working for you now?

**Comment by ccordoba12** [comment]
Great! Thanks for letting us know about it.
